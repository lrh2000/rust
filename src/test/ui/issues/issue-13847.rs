fn main() {
    return.is_failure //~ ERROR type annotations needed
}
