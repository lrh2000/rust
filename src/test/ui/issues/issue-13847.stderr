error[E0282]: type annotations needed
  --> $DIR/issue-13847.rs:2:5
   |
LL |     return.is_failure
   |     ^^^^^^ cannot infer type
   |
   = note: type must be known at this point

error: aborting due to previous error

For more information about this error, try `rustc --explain E0282`.
